standards.md

```diff
@@ -149,6 +169,20 @@ requires developers to follow more links rather than just showing
 what we want.
 ```
 
+## Small Pull Requests
+
+* Small pull requests that make a single, self-contained change are easier to review and easier to roll back or backport.
+  * If you can think of a way to break up a pull request into smaller changes, do it
+* Tests and documentation (where applicable) should always be part of the same pull request when introducing a change
+* Use one commit per PR unless there is a strong reason not to
+  * For multiple commits, ensure each makes sense without the context of the others.
+* Refactoring should be merged before bug fixes and features
+  * i.e. if you refactor as part of implementing something, commit it and merge it before merging the change
+* Features may be implemented incrementally to keep pull requests small
+  * When introducing a partial feature, the documentation should include updates that
+    indicate clearly that this functionality is not expected to work and point the reader
+    toward how to follow progress (e.g. via an issue)
+
```

Comment:
suggest giving an example commit message for this

Reply:
Agree that it's a bit overspecified -- but I think it's helpful to overspecify for people new to this "small PR" principle in development.

/LGTM